Microsoft.Toolkit.Win32.UI.Controls/Interop/WinRT/WebViewControlProcess.cs

```diff
@@ -0,0 +1,144 @@
+// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Security;
+using System.Threading.Tasks;
+
+using Windows.Foundation;
+using Windows.Web.UI.Interop;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT
+{
+
+    /// <summary>
+    /// A proxy for <see cref="Windows.Web.UI.Interop.WebViewControlProcess"/>.
+    /// </summary>
+    public class WebViewControlProcess
```

Comment:
@normesta @mcleanbyron Most documentation for existing WinRT types was taken from the docs site